README.md

```diff
@@ -1,20 +1,118 @@
+
+```html
+<ff-predictive-basket user-id="123456"
+                      max-results="10"
+                      certainty="60"
+                      blacklist="art-id1,art-id2">
+    <!-- requires ff-record-list here to work -->
+</ff-predictive-basket>
+```
+
+Once the element is correctly initialized, it will automatically issue a request to FACT-Finder and display the result. This will happen only once per page load. If you need to refresh the displayed data, refer to the `getPredictions()` method below.
+
+#### Attributes
+
+Attribute | Required | Type | Default | JS property
+--------- | -------- | ---- | ------- | -----------
+user-id   | yes      | String | _(none)_ | `userId`
+
+The ID of the user for whom products shall be predicted. If the user ID is not specified, no request to FACT-Finder will be issued.
+
+
+
+Attribute | Required | Type | Default | JS property
+--------- | -------- | ---- | ------- | -----------
+max-results | optional | Number |  _(unlimited)_ | `maxResults`
+
+The maximum amount of products to be returned from FACT-Finder. To receive all predictions, omit this attribute or set its value to `-1`.
+
+
+Attribute | Required | Type | Default | JS property
+--------- | -------- | ---- | ------- | -----------
+certainty | optional | Number | 60    | `certainty`
+
+This value is the minimum percentage to which the prediction algorithm is certain that the user will buy the product. Products with a lower certainty will not be returned from FACT-Finder.
+
+
+Attribute | Required | Type | Default | JS property
+--------- | -------- | ---- | ------- | -----------
+blacklist | optional | String (comma-separated list) | _(empty)_ | `blacklist`
+
+With the `blacklist` attribute you can specify one or more product IDs that shall not be returned from FACT-Finder. This may be useful if you, for example, don't want products to appear in the predictions once the user has placed them in their shopping cart.
+
+#### Methods
+
+##### `predictiveBasketElement.getPredictions()`
+
+This method issues a request to FACT-Finder with the currently set values.
+
+
+### Example Setup
+
+#### HTML
+```html
+<ff-predictive-basket user-id="123456" max-results="10" certainty="80" blacklist="art-id1,art-id2">
+    <ff-record-list subscribe="false">
+        <ff-record>
+            <img data-image="{{record.image}}" data-image-onerror="../img_not_found.gif">
+            <div>
+                <a data-anchor="{{record.deeplink}}" data-redirect="{{record.deeplink}}">
+                    {{record.name}}
+                </a>
+                <div>Price: <strong>{{record.price}}</strong></div>
+            </div>
+        </ff-record>
+    </ff-record-list>
+</ff-predictive-basket>
+```
+
+#### Manual update with `blacklist`
+```js
+function onUpdate(itemsToExclude) {
+    const predBasket = document.querySelector("ff-predictive-basket");
+    predBasket.blacklist = itemsToExclude;  // itemsToExclude must be a comma-separated list
```

Comment:
```suggestion
    predBasket.blacklist = ["blacklistedId1", "blacklistedId2", "blacklistedId3"];  // itemsToExclude must be a comma-separated list
```

Reply:
doesnt make sense in the context of this function x/ but think showing the the data type of itemsToExclude more verbose makes sense.